test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Great job implementing tests! 

You should definitely look into using RSpec next time! 👍